public/js/components/AtomEdit/CustomEditors/StoryQuestionFields/Answer.js

```diff
@@ -0,0 +1,48 @@
+import React, { PropTypes } from 'react';
+import {ManagedForm, ManagedField} from '../../../ManagedEditor';
+import FormFieldTextInput from '../../../FormFields/FormFieldTextInput';
+import SearchSelectBox from '../../../FormFields/SearchFields/SearchSelectBox';
+
+export class StoryQuestionsAnswer extends React.Component {
+  static propTypes = {
+    fieldLabel: PropTypes.string,
+    fieldName: PropTypes.string,
+    fieldValue: PropTypes.shape({
+      answerId: PropTypes.string,
+      answerType: PropTypes.string
+    }),
+    fieldPlaceholder: PropTypes.string,
+    onUpdateField: PropTypes.func,
+    onFormErrorsUpdate: PropTypes.func
+  };
+
+  onUpdate = (answer) => {
+    const answerWithType = Object.assign({}, answer, {
+      answerType: answer.answerType || "ATOM"
+    });
+
+    this.props.onUpdateField(answerWithType);
+  }
+
+  getPlaceholder = (answerType) => answerType === "CONTENT"
```

Comment:
What's the logic behind the values for the placeholders here?

Reply:
they're just examples of a content ID and atom ID